.github/workflows/build.yaml

```diff
@@ -26,6 +26,13 @@ jobs:
     - name: Type check with typescript
       run: |
         npm run type-check
+
     - name: Test with jest
       run: |
-        npm test test/unit
+        npm test -- --coverage test/unit
+        
+    - name: "Security Scan"
```

Comment:
Do we want to make this a part of build? Do we have any other cases where we want to do security scan without build?
Maybe we should make it depend on build in case it's required for scan?

Reply:
SonarCloud unit test coverage requires build in order to be calculated so it would always require build. Now, the opposite question is also valid, does build require sonarcloud to run all the time? Looking through the other GH workflows in the repo, build is not reused. Release workflow actually does a `build` step, which is different from this one. I'd argue this workflow should be named "PR checks", similar to what we have in Python SDK, and left as is.